deploy/grafana-service.yaml

```diff
@@ -1,7 +1,13 @@
-apiVersion: "v1beta1"
-kind: "Service"
-id: "monitoring-grafana"
-port: 80
-containerPort: 80
-selector: 
-  name: "influxGrafana"
+apiVersion: v1beta3
+kind: Service
+metadata: 
+  labels: 
+    name: influxGrafana
```

Comment:
Isn't it a good practice to have the selector as a label in the service too, I saw it in the  discussions on some PR